### DISCUSSION TITLE
Load process.env in boot file

### DISCUSSION THREAD

**Discussion by mJeromeDiaz** [quasarframework/quasar#12991] [General - Components / Directives / etc] [answered]
Hi all,
I am trying to edit my axios boot file.
I want to create a baseURL dynamically depending on whether I'm in development or in production.
I stored my variable in quasar.conf.js, but I can't import this variable into my boot file.

```js
    build: {
      vueRouterMode: "hash", // available values: 'hash', 'history'
      env: ctx.dev ? {
        API_URL: 'https://127.0.0.1:8000/'
      } : {
        API_URL: 'https://domain.com/api/'
      },

```

```js
export default boot(({ app, api }) => {
  axios.defaults.baseURL = process.env.API_URL;
  axios.defaults.timeout = 1500;
  axios.interceptors.request.use(config => {

```

But i cant import process...

**Comment by yusufkandemir** [comment, marked as answer]
You don't need to "import" anything. Just typing `process.env.API_URL` is enough.
If you have an error, please share it along with the output of the `quasar info` command.

**Reply by mJeromeDiaz** [reply]
Thank you :)
